Started by timer
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Still waiting to schedule task
Waiting for next available executor on â€˜[8mha:////4C8PgMHPegRpCg8+HpIiIHK02uxvuQktJylkqRHkJagNAAAAox+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzeEgYJ/eT83ILSktQiICOvpCg/Jye1SNfSUh8ABjlyb80AAAA=[0mcontroller-99â€™
Running on [8mha:////4C8PgMHPegRpCg8+HpIiIHK02uxvuQktJylkqRHkJagNAAAAox+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzeEgYJ/eT83ILSktQiICOvpCg/Jye1SNfSUh8ABjlyb80AAAA=[0mcontroller-99 in /root/.jenkins/workspace/icx-d-test-nic-sriov-5times
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (rm project dir)
[8mha:////4EWjgIQS1R18e39T6LdOu3ZpSMSXMburmKwxJHd8e163AAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gwExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jcChmMxwAAAAA==[0m[Pipeline] script
[8mha:////4NOJ/YVWDlnnBpjDdLKlLZkF5LrR387vblsx7fWhZkPlAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0BAkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfpX/cvwAAAAA==[0m[Pipeline] {
[8mha:////4MAm0moKTc8Eo2PcB0LgkHNHI7Xtf5Poej2Mf/l8xh/lAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaoyVLaHxBMgiwpL/WWDdrTyRV/MOEjexcpJJ5k3zXm9Y5wQnSpZ7g73DrB2PYbR18YlSfws0cU9XrgkzBcOlmSR15rygpGJgCWtgJWBjUAfKDm2BrfDqodqg0LaXkup3FNC4boAnMFHNRaXyhTnB7j52mfDn+SuAORZg+9pD/AAeoCAqvAAAAA==[0m[Pipeline] sh
+ ls /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester
1.0.0.0001.version
conf
installation
libs
log_analyzer
README.md
testconf
testLibs
Tests
[8mha:////4F3BY6hez6Yn9lSF4dYULOlz4RwK0Fdsp1OYWtj0vxDtAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKBDglR0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYO0G2F+S7bfcBITL9lL0AAAA=[0m[Pipeline] echo
dir is exists and rm dir
[8mha:////4KL1cTFlC/t8/0LRf71hJwxXIUFLIMviWtcIHXR2W8epAAAAox+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6KEzcQ4uTYsfkGFWgvNe9g+hMkv8tf8B4kkTt7pnrOc1xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9YJsL8jLsv8AYAPmjb0AAAA=[0m[Pipeline] sh
+ rm -rf /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester
[8mha:////4GzJynz9IX3CLl34e/1JpYOeLz7FDsqMtIihCtFGqNpZAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRVEiIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPzjQzob0AAAA=[0m[Pipeline] }
[8mha:////4MvApYhNLg1xEFG2iXyHCog7Z9h4MbdAsNSDoK4wJnDgAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRoqBAVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AO8A7Ay9AAAA[0m[Pipeline] // script
[8mha:////4P1g47DHQeYIMdAQPrqVFJW5XTEYSJkMqP0k+p1upCWxAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIREqJBVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AGnsy8m9AAAA[0m[Pipeline] }
[8mha:////4Lc6WrLYEiiAC7rTWY8q4RQPGu1FD+hrgClvEqpzcg2sAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiKMBZVjZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0g35eCrPwAY5Sibb0AAAA=[0m[Pipeline] // stage
[8mha:////4LPzYIRsJlP9vBl87WR2FbvV5590n9jZkqHCRBijPEs5AAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjJESDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A6sHc4fBAAAA[0m[Pipeline] stage
[8mha:////4P0yjWfQiEwQxZO7i9obwDXrYwQclNR3i//1CTGbFwBFAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohULDAgpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD1+IXOLCAAAA[0m[Pipeline] { (git clone)
[8mha:////4I3tKNYAz7At397ZkNOs6SFwEn2bzn3ZBhsoegW1dhRwAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKpAQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHNxXfVKy+AAAA[0m[Pipeline] sh
+ git clone -b develop https://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@github.com/intel-innersource/applications.validation.network-platform.npxtester.git npxtester
Cloning into 'npxtester'...
[8mha:////4CRWXV91kUhqh2bTVXI+2n5U32cQ89nJ7yub5blKQ29bAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKoQQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrt/lO7N9Jjga++AAAA[0m[Pipeline] }
[8mha:////4I4ZVrOFMvh2sVfxnfT9s6+paGHoH9OzkxEwdR4h+nRDAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICFDQoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDQMEcr9Ntfu8AERGGHOvgAAAA==[0m[Pipeline] // stage
[8mha:////4J4BpfnRrLE6odVrNZGgZ11wIRy0HYHBalvcHAcD12j/AAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjEB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INji2eq8EAAAA=[0m[Pipeline] stage
[8mha:////4AgPhGmio4pMNzq5gItmuDxswb3P5dwyIofUIBdMXBTmAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+DOrAycohUdEVMrFEXThCaEJJG/7dJSjpxIq7GHWipxIQHy7ZkvdcbyhjgRMFwp7GzGFvLez+aOfFMobt5ytzRlbeEkbzmjc4NKX1ea0NJw6qCwUbAVmPrKVo0CXbCyYesvERTXVKYt6MAZtUATyjETE4ypG+ZAuzvo4qEP85fAEx9AlYfFquXZ/kBAsTc3cIAAAA=[0m[Pipeline] { (pre-exec-set-args)
[8mha:////4G9dxluK5ut2lxdfo2JVi6OIov1ZH/881VUFzuPzanonAAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggyE0vHypah8QSYIEKY/wkQSeWJvJp3EM2MlVvs7NvmPV+wjgGOFDSzCgeDsTPMu0mXxTKF4eooM0sX1hFGcooJlQX16rSgoKRgSVXDisNGYecoGtQJttzKu2ycRN2cUyjfgUNt+hEeUPFiTjKkL8wBdrepj4Q/z18BzD5B3baf2vs3w5BRkb4AAAA=[0m[Pipeline] sh
+ python /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/conf/change_args.py '{"drive_info":{"drive_version":"1.7.19"},"dpdk_dir":"/opt/APP/utility/DPDK/dpdk-21.11/","vm_image_conf":{"image1":"uvm1.img","image2":"uvm2.img","image3":"rhel84rt.img","image4":"rhel84rt2.img","image_dir":"/home/image"},"sut_conf":{"host":"10.239.115.71","pf1":"enp137s0f0","os":"UBUNTU"},"pkg_generator":{"host":"10.239.115.16","pf1":"ens801f0"}}'
[8mha:////4FhUQONKPXFSeOjFO92QKbtxv93cUzSuhB7hx3xoWX6KAAAApB+LCAAAAAAAAP9tjbEOwiAURV9rHFwd/QiaNG7GyZWw+AXYIkLJexSodPKL/DX/QbSJk3e4uecu5/mCdQxwpKCZVTgYjJ1h3k26LJYpDFdHmVm6sI4wklNMqCyoV6cFBSUFS6oaVhw2CjtH0aBOsOVW3mXjJOrmnEL5Dhxq04/wgIoXc5IhfWEOsLtNfST8ef4KYPY+Qd3uP9W+Ad3NsHm+AAAA[0m[Pipeline] }
[8mha:////4EpM1Tihe33kOTBucrClpdwMCq1FF6wGwTU/AEOZmYaNAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikKgQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrd/lPbN8JP5Su+AAAA[0m[Pipeline] // stage
[8mha:////4JJk4NQMkVX21N+ktknOqKLQw+FD2VU/fvD8dhBPuambAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjIUSDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8AzYdm7bBAAAA[0m[Pipeline] stage
[8mha:////4GToWcz1lEtttcc1F67+obVKf97ChyGWoNBgdNb8ln6iAAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYoABMXWNsnCC0ISQNvq//KS0EyfiatyBlkpMeLBsS9Z7vaGIDCdiJxqLrcdYe9GF3k1JDMTtNdAgGrqImjBSsELZQZGx1VIVJQuLshxWEtYW60DRo0uwkY1+6DJodOU58bQdJeTe3OEJmZzISXP6lpFhe+tNJPxx/gJg7BLku/1sh/lZfAAyQCqkwgAAAA==[0m[Pipeline] { (run nic case)
[8mha:////4M/BBJOrvLliK1a64sS7A0hf4H4seiclg8D1IX9OqKAVAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0YCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDhnK7+9Q+vAGvZp0gvgAAAA==[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics -s -m nicSriov5times --capture=tee-sys --html=/root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-2.1.1
collected 20 items / 19 deselected / 1 selected

npxtester/Tests/testNics/test_network_sriov_stress.py 2022-05-03 03:09:39 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-05-03/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm2.img-uvm1.img]_03_09_39/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm2.img-uvm1.img].log, case_logdir: /root/testconfig/testlogs/2022-05-03/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm2.img-uvm1.img]_03_09_39
2022-05-03 03:09:39 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm2.img-uvm1.img] case ============================
2022-05-03 03:09:39 - test_network_sriov_stress.py - [line:32] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-05-03 03:09:39 - test_network_sriov_stress.py - [line:50] - root - INFO - ssh to the sut1
2022-05-03 03:09:39 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-05-03 03:09:39 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (publickey) failed.
2022-05-03 03:09:39 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-05-03 03:09:39 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-05-03 03:09:39 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-05-03 03:09:39 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (publickey) failed.
2022-05-03 03:09:39 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-05-03 03:09:39 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.16
2022-05-03 03:09:39 - test_network_sriov_stress.py - [line:64] - root - INFO - clean up iperf and taskset process...
2022-05-03 03:09:42 - sshInstance.py - [line:50] - root - INFO - pkill -9 iperf
2022-05-03 03:09:42 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:09:50 - sshInstance.py - [line:50] - root - INFO - pkill -9 taskset
2022-05-03 03:09:50 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:09:58 - sshInstance.py - [line:50] - root - INFO - pkill -9 iperf
2022-05-03 03:09:58 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:10:06 - sshInstance.py - [line:50] - root - INFO - pkill -9 taskset
2022-05-03 03:10:07 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:10:12 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 3] Opened sftp connection (server version 3)
2022-05-03 03:10:12 - sshInstance.py - [line:201] - root - INFO - start to rmmod irdma....
2022-05-03 03:10:15 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-05-03 03:10:15 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-05-03 03:10:20 - sshInstance.py - [line:204] - root - INFO - finished rmmod irdma....
2022-05-03 03:10:20 - sshInstance.py - [line:205] - root - INFO - start to rmmod ice....
2022-05-03 03:10:23 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-05-03 03:10:25 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:10:30 - sshInstance.py - [line:208] - root - INFO - finished rmmod ice....
2022-05-03 03:10:30 - sshInstance.py - [line:210] - root - INFO - start to modprobe ice....
2022-05-03 03:10:33 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-05-03 03:10:38 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:10:43 - sshInstance.py - [line:212] - root - INFO - finished modprobe ice....
2022-05-03 03:10:46 - sshInstance.py - [line:50] - root - INFO - ethtool -i enp137s0f0
2022-05-03 03:10:46 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.7.19
firmware-version: 2.19 0x8000bafe 1.3083.0
expansion-rom-version: 
bus-info: 0000:89:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-05-03 03:10:51 - test_network_sriov_stress.py - [line:71] - root - INFO - try to get ipv6 for enp137s0f0
2022-05-03 03:10:51 - test_network_sriov_stress.py - [line:77] - root - INFO - try to get ipv6 of pf for ubuntu...
2022-05-03 03:10:54 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 up
2022-05-03 03:10:54 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:10:59 - test_network_sriov_stress.py - [line:80] - root - INFO - try to get ipv6 for ens801f0
2022-05-03 03:10:59 - test_network_sriov_stress.py - [line:82] - root - INFO - getting ipv6 in redhat.
2022-05-03 03:10:59 - sshInstance.py - [line:188] - root - INFO - starting to delete network of ens801f0:
2022-05-03 03:11:05 - sshInstance.py - [line:99] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Tue May  3 02:56:16 2022 from 10.239.182.99
[root@redhat8 ~]# nmcli connection delete ens801f0
Connection 'ens801f0' (bb122480-1bf8-4bc9-ba66-35a7fba716cf) successfully deleted.
[root@redhat8 ~]# 
[root@redhat8 ~]# 
2022-05-03 03:11:08 - sshInstance.py - [line:190] - root - INFO - restart networkManager to make setting valid
2022-05-03 03:11:14 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@redhat8 ~]# 
[root@redhat8 ~]# 
2022-05-03 03:11:17 - sshInstance.py - [line:194] - root - INFO - starting to add network of ens801f0:
2022-05-03 03:11:23 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens801f0 ifname enss801f0
Connection 'ens801f0' (c4648774-2579-4f1b-9a03-eb8bc41049fc) successfully added.
[root@redhat8 ~]# 
[root@redhat8 ~]# 
2022-05-03 03:11:26 - sshInstance.py - [line:197] - root - INFO - restart networkManager to make setting valid
2022-05-03 03:11:32 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@redhat8 ~]# 
[root@redhat8 ~]# 
2022-05-03 03:11:35 - test_network_sriov_stress.py - [line:90] - root - INFO - trying to get ipv6 addr for pf1
2022-05-03 03:11:38 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0
2022-05-03 03:11:38 - sshInstance.py - [line:54] - root - INFO - enp137s0f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::200:ff:fe00:3300  prefixlen 64  scopeid 0x20<link>
        ether 00:00:00:00:33:00  txqueuelen 1000  (Ethernet)
        RX packets 6  bytes 1962 (1.9 KB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 14  bytes 2451 (2.4 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-05-03 03:11:46 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0
2022-05-03 03:11:46 - sshInstance.py - [line:54] - root - INFO - ens801f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::921b:cedb:cf61:b184  prefixlen 64  scopeid 0x20<link>
        ether 3c:fd:fe:d2:73:e0  txqueuelen 1000  (Ethernet)
        RX packets 178230820623  bytes 269826402183544 (245.4 TiB)
        RX errors 0  dropped 973952  overruns 0  frame 0
        TX packets 9841580424  bytes 718875876824 (669.5 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-05-03 03:11:51 - test_network_sriov_stress.py - [line:621] - root - INFO - 
Stress Disable and Enable for 1 time when VF is Assigned VM guest.

2022-05-03 03:11:54 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -s | grep enp137s0f0 | awk '{print $1;}'
2022-05-03 03:11:54 - sshInstance.py - [line:54] - root - INFO - 0000:89:00.0

2022-05-03 03:11:59 - test_network_sriov_stress.py - [line:156] - root - INFO - pf_id was found: 0000:89:00.0

2022-05-03 03:11:59 - test_network_sriov_stress.py - [line:629] - root - INFO - format_pf_id is 0000\:89\:00.0

2022-05-03 03:11:59 - test_network_sriov_stress.py - [line:632] - root - INFO - pf_id_key: 89
2022-05-03 03:11:59 - test_network_sriov_stress.py - [line:636] - root - INFO - start to assign ip for enp137s0f0 of SUT1: 192.168.10.10
2022-05-03 03:12:05 - sshInstance.py - [line:99] - root - INFO - Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
Last login: Tue May  3 03:13:06 2022 from 10.239.182.99
root@npx-sut3-ice-d-auto:~# ifconfig enp137s0f0 192.168.10.10
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:12:08 - test_network_sriov_stress.py - [line:365] - root - INFO - Successfully assigned ip to ethernet: enp137s0f0
2022-05-03 03:12:08 - test_network_sriov_stress.py - [line:638] - root - INFO - start to assign ip for ens801f0 of SUT2: 192.168.10.11
2022-05-03 03:12:14 - sshInstance.py - [line:99] - root - INFO - ifconfig ens801f0 192.168.10.11
[root@redhat8 ~]# 
2022-05-03 03:12:17 - test_network_sriov_stress.py - [line:365] - root - INFO - Successfully assigned ip to ethernet: ens801f0
2022-05-03 03:12:17 - test_network_sriov_stress.py - [line:162] - root - INFO - virtulize sut1 pf to 4 vfs
2022-05-03 03:12:20 - sshInstance.py - [line:50] - root - INFO - echo 4 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-05-03 03:12:21 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:12:26 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfully Virtualized PF to VFs.
2022-05-03 03:12:29 - sshInstance.py - [line:50] - root - INFO - lspci | grep "Virtual" |grep 89 | awk '{print $1;}'list
2022-05-03 03:12:29 - sshInstance.py - [line:54] - root - INFO - 89:01.0
89:01.1
89:01.2
89:01.3

2022-05-03 03:12:34 - test_network_sriov_stress.py - [line:185] - root - INFO - Succssfully Gotten vf_id list:

2022-05-03 03:12:37 - sshInstance.py - [line:50] - root - INFO - modprobe vfio-pci

2022-05-03 03:12:37 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:12:42 - test_network_sriov_stress.py - [line:205] - root - INFO - start to bind drive in sut1
2022-05-03 03:12:45 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.0
2022-05-03 03:12:45 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:12:50 - test_network_sriov_stress.py - [line:219] - root - INFO - Successfuly bound VF to vfio-pci
2022-05-03 03:12:53 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.1
2022-05-03 03:12:54 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:12:59 - test_network_sriov_stress.py - [line:219] - root - INFO - Successfuly bound VF to vfio-pci
2022-05-03 03:12:59 - test_network_sriov_stress.py - [line:656] - root - INFO - lspci -Dnn | grep 89:01.1
2022-05-03 03:13:02 - sshInstance.py - [line:50] - root - INFO - lspci -Dnn | grep 89:01.1
2022-05-03 03:13:02 - sshInstance.py - [line:54] - root - INFO - 0000:89:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889]

2022-05-03 03:13:07 - test_network_sriov_stress.py - [line:668] - root - INFO - Successfully get vf device id: 8086:1889
2022-05-03 03:13:07 - test_network_sriov_stress.py - [line:242] - root - INFO - get default network info: virsh net-info default | grep "Active" | awk '{print $2;}'
2022-05-03 03:13:10 - sshInstance.py - [line:50] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-05-03 03:13:10 - sshInstance.py - [line:54] - root - INFO - yes

2022-05-03 03:13:15 - test_network_sriov_stress.py - [line:245] - root - INFO - Default network is active? yes

2022-05-03 03:13:15 - test_network_sriov_stress.py - [line:681] - root - INFO - start to create VMs....
2022-05-03 03:13:15 - test_network_sriov_stress.py - [line:275] - root - INFO - try to create VM with the cmd1: virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk path=/home/image/uvm2.img,size=50,format=qcow2 --osinfo ubuntu20.04 --graphics vnc,listen=0.0.0.0,port=5900 --noautoconsole

2022-05-03 03:13:19 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --osinfo ubuntu20.04 --graphics vnc,,listen=0.0.0.0,port=5900 --noautoconsole
usage: virt-install --name NAME --memory MB STORAGE INSTALL [options]
virt-install: error: unrecognized arguments: --osinfo ubuntu20.04
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:13:22 - test_network_sriov_stress.py - [line:278] - root - INFO - cmd1 is not working, try to create VM with the cmd2: virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk path=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900 --noautoconsole

2022-05-03 03:13:26 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-05-03 03:13:29 - sshInstance.py - [line:75] - root - INFO - Domain creation completed.
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:13:32 - test_network_sriov_stress.py - [line:292] - root - INFO - Successfully created a VM: VMtest1
2022-05-03 03:13:32 - test_network_sriov_stress.py - [line:275] - root - INFO - try to create VM with the cmd1: virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk path=/home/image/uvm1.img,size=50,format=qcow2 --osinfo ubuntu20.04 --graphics vnc,listen=0.0.0.0,port=5901 --noautoconsole

2022-05-03 03:13:36 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --osinfo ubuntu20.04 --graphics vnc,,listen=0.0.0.0,port=5901 --noautoconsole
usage: virt-install --name NAME --memory MB STORAGE INSTALL [options]
virt-install: error: unrecognized arguments: --osinfo ubuntu20.04
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:13:39 - test_network_sriov_stress.py - [line:278] - root - INFO - cmd1 is not working, try to create VM with the cmd2: virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk path=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901 --noautoconsole

2022-05-03 03:13:43 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-05-03 03:13:46 - sshInstance.py - [line:75] - root - INFO - Domain creation completed.
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:13:49 - test_network_sriov_stress.py - [line:292] - root - INFO - Successfully created a VM: VMtest2
2022-05-03 03:14:23 - sshInstance.py - [line:50] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-05-03 03:14:23 - sshInstance.py - [line:54] - root - INFO - 192.168.122.228/24

2022-05-03 03:14:28 - test_network_sriov_stress.py - [line:308] - root - INFO - Successfully got VMtest1 IP: 192.168.122.228/24

2022-05-03 03:14:31 - sshInstance.py - [line:50] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-05-03 03:14:31 - sshInstance.py - [line:54] - root - INFO - 192.168.122.92/24

2022-05-03 03:14:36 - test_network_sriov_stress.py - [line:308] - root - INFO - Successfully got VMtest2 IP: 192.168.122.92/24

2022-05-03 03:14:36 - test_network_sriov_stress.py - [line:692] - root - INFO - ip of VM1: 192.168.122.228
2022-05-03 03:14:36 - test_network_sriov_stress.py - [line:693] - root - INFO - ip of VM1: 192.168.122.92
2022-05-03 03:14:36 - test_network_sriov_stress.py - [line:698] - root - INFO - Start ssh VM1 and conf VM1
2022-05-03 03:14:42 - sshInstance.py - [line:99] - root - INFO - ssh-key -R 192.168.122.228
ssh-key: command not found
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:14:49 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.122.228
PING 192.168.122.228 (192.168.122.228) 56(84) bytes of data.
64 bytes from 192.168.122.228: icmp_seq=1 ttl=64 time=0.088 ms
2022-05-03 03:14:50 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.228: icmp_seq=2 ttl=64 time=0.075 ms
2022-05-03 03:14:51 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.228: icmp_seq=3 ttl=64 time=0.073 ms
2022-05-03 03:14:52 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.228: icmp_seq=4 ttl=64 time=0.071 ms
2022-05-03 03:14:53 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.228: icmp_seq=5 ttl=64 time=0.071 ms

--- 192.168.122.228 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4092ms
rtt min/avg/max/mdev = 0.071/0.075/0.088/0.006 ms
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:15:02 - sshInstance.py - [line:99] - root - INFO - ssh root@192.168.122.228
root@192.168.122.228's password: 
2022-05-03 03:15:11 - sshInstance.py - [line:99] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Mon 02 May 2022 03:18:48 PM EDT
  System load:  0.07               Processes:             136
  Usage of /:   40.4% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.228
  Swap usage:   0%

383 updates can be installed immediately.
261 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun May  1 20:34:55 2022 from 192.168.122.1
root@localhost:~# 
2022-05-03 03:15:14 - test_network_sriov_stress.py - [line:392] - root - INFO - Successfully ssh VM.
2022-05-03 03:15:20 - sshInstance.py - [line:99] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-05-03 03:15:29 - test_network_sriov_stress.py - [line:414] - root - INFO - successfully got vf pci id of vm: 0000:00:05.0
2022-05-03 03:15:35 - sshInstance.py - [line:99] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-05-03 03:15:40 - test_network_sriov_stress.py - [line:428] - root - INFO - Successfully got vf device name of vm: ens5
2022-05-03 03:15:40 - test_network_sriov_stress.py - [line:440] - root - INFO - try to get ipv6 for ens5
2022-05-03 03:15:40 - test_network_sriov_stress.py - [line:446] - root - INFO - try to get ipv6 in ubuntu
2022-05-03 03:15:46 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-05-03 03:15:55 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::d40d:bcff:fe08:f023  prefixlen 64  scopeid 0x20<link>
        ether d6:0d:bc:08:f0:23  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 331 (331.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 1  bytes 70 (70.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-05-03 03:15:58 - test_network_sriov_stress.py - [line:455] - root - INFO - assign ip address for VM...
2022-05-03 03:16:04 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-05-03 03:16:07 - test_network_sriov_stress.py - [line:365] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-05-03 03:16:13 - sshInstance.py - [line:99] - root - INFO - exit
logout
Connection to 192.168.122.228 closed.
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:16:16 - test_network_sriov_stress.py - [line:704] - root - INFO - Start ssh VM2 and conf VM2
2022-05-03 03:16:22 - sshInstance.py - [line:99] - root - INFO - ssh-key -R 192.168.122.92
ssh-key: command not found
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:16:29 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.122.92
PING 192.168.122.92 (192.168.122.92) 56(84) bytes of data.
64 bytes from 192.168.122.92: icmp_seq=1 ttl=64 time=0.086 ms
2022-05-03 03:16:30 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.92: icmp_seq=2 ttl=64 time=0.078 ms
2022-05-03 03:16:31 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.92: icmp_seq=3 ttl=64 time=0.072 ms
2022-05-03 03:16:32 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.92: icmp_seq=4 ttl=64 time=0.074 ms
2022-05-03 03:16:33 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.92: icmp_seq=5 ttl=64 time=0.078 ms

--- 192.168.122.92 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4090ms
rtt min/avg/max/mdev = 0.072/0.077/0.086/0.004 ms
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:16:42 - sshInstance.py - [line:99] - root - INFO - ssh root@192.168.122.92
The authenticity of host '192.168.122.92 (192.168.122.92)' can't be established.
ECDSA key fingerprint is SHA256:uiL9MQ8j64kizIc2NkvuxVE93C2YBfGXw5p1QHW5dUA.
Are you sure you want to continue connecting (yes/no/[fingerprint])? 
2022-05-03 03:16:51 - sshInstance.py - [line:99] - root - INFO - yes
Warning: Permanently added '192.168.122.92' (ECDSA) to the list of known hosts.
root@192.168.122.92's password: 
2022-05-03 03:17:00 - sshInstance.py - [line:99] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Mon 02 May 2022 03:20:37 PM EDT
  System load:  0.01               Processes:             134
  Usage of /:   41.1% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.92
  Swap usage:   0%

382 updates can be installed immediately.
265 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun May  1 20:33:15 2022 from 192.168.122.1
root@localhost:~# 
2022-05-03 03:17:03 - test_network_sriov_stress.py - [line:392] - root - INFO - Successfully ssh VM.
2022-05-03 03:17:09 - sshInstance.py - [line:99] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-05-03 03:17:18 - test_network_sriov_stress.py - [line:414] - root - INFO - successfully got vf pci id of vm: 0000:00:05.0
2022-05-03 03:17:24 - sshInstance.py - [line:99] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-05-03 03:17:29 - test_network_sriov_stress.py - [line:428] - root - INFO - Successfully got vf device name of vm: ens5
2022-05-03 03:17:29 - test_network_sriov_stress.py - [line:440] - root - INFO - try to get ipv6 for ens5
2022-05-03 03:17:29 - test_network_sriov_stress.py - [line:446] - root - INFO - try to get ipv6 in ubuntu
2022-05-03 03:17:35 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-05-03 03:17:44 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::989e:9ff:fe44:a64d  prefixlen 64  scopeid 0x20<link>
        ether 9a:9e:09:44:a6:4d  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 1  bytes 70 (70.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-05-03 03:17:47 - test_network_sriov_stress.py - [line:455] - root - INFO - assign ip address for VM...
2022-05-03 03:17:53 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-05-03 03:17:56 - test_network_sriov_stress.py - [line:365] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-05-03 03:18:02 - sshInstance.py - [line:99] - root - INFO - exit
logout
Connection to 192.168.122.92 closed.
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:18:05 - test_network_sriov_stress.py - [line:712] - root - INFO - Ping two vms IP from SUT:
2022-05-03 03:18:09 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.347 ms
2022-05-03 03:18:10 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.062 ms
2022-05-03 03:18:11 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.071 ms
2022-05-03 03:18:12 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.044 ms
2022-05-03 03:18:13 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.042 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4090ms
rtt min/avg/max/mdev = 0.042/0.113/0.347/0.117 ms
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:18:20 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.255 ms
2022-05-03 03:18:21 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.054 ms
2022-05-03 03:18:22 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.042 ms
2022-05-03 03:18:23 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.036 ms
2022-05-03 03:18:24 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.035 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4084ms
rtt min/avg/max/mdev = 0.035/0.084/0.255/0.085 ms
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:18:27 - test_network_sriov_stress.py - [line:720] - root - INFO - ping VMtest2 IP from VMtest1
2022-05-03 03:18:33 - sshInstance.py - [line:99] - root - INFO - ssh-key -R 192.168.122.228
ssh-key: command not found
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:18:40 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.122.228
PING 192.168.122.228 (192.168.122.228) 56(84) bytes of data.
64 bytes from 192.168.122.228: icmp_seq=1 ttl=64 time=0.104 ms
2022-05-03 03:18:41 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.228: icmp_seq=2 ttl=64 time=0.079 ms
2022-05-03 03:18:42 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.228: icmp_seq=3 ttl=64 time=0.081 ms
2022-05-03 03:18:43 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.228: icmp_seq=4 ttl=64 time=0.085 ms
2022-05-03 03:18:44 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.228: icmp_seq=5 ttl=64 time=0.074 ms

--- 192.168.122.228 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4090ms
rtt min/avg/max/mdev = 0.074/0.084/0.104/0.010 ms
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:18:53 - sshInstance.py - [line:99] - root - INFO - ssh root@192.168.122.228
root@192.168.122.228's password: 
2022-05-03 03:19:02 - sshInstance.py - [line:99] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Mon 02 May 2022 03:22:40 PM EDT
  System load:  0.0                Processes:             139
  Usage of /:   40.4% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.228
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

383 updates can be installed immediately.
261 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Mon May  2 15:18:49 2022 from 192.168.122.1
root@localhost:~# 
2022-05-03 03:19:05 - test_network_sriov_stress.py - [line:392] - root - INFO - Successfully ssh VM.
2022-05-03 03:19:09 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.156 ms
2022-05-03 03:19:10 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.045 ms
2022-05-03 03:19:11 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.031 ms
2022-05-03 03:19:12 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.038 ms
2022-05-03 03:19:13 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.025 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4093ms
rtt min/avg/max/mdev = 0.025/0.059/0.156/0.048 ms
root@localhost:~# 
2022-05-03 03:19:22 - sshInstance.py - [line:99] - root - INFO - exit
logout
Connection to 192.168.122.228 closed.
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:19:25 - test_network_sriov_stress.py - [line:728] - root - INFO - ping VMtest2 IP from VMtest1
2022-05-03 03:19:29 - sshInstance.py - [line:75] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.114 ms
2022-05-03 03:19:30 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.043 ms
2022-05-03 03:19:31 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.039 ms
2022-05-03 03:19:32 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.037 ms
2022-05-03 03:19:33 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.039 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4081ms
rtt min/avg/max/mdev = 0.037/0.054/0.114/0.029 ms
root@npx-sut3-ice-d-auto:~# 
2022-05-03 03:19:39 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-05-03 03:19:42 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:19:47 - test_network_sriov_stress.py - [line:327] - root - INFO - Successfully removed VFs from PF.
2022-05-03 03:19:50 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest1
2022-05-03 03:19:50 - sshInstance.py - [line:54] - root - INFO - Domain VMtest1 destroyed


2022-05-03 03:19:55 - test_network_sriov_stress.py - [line:340] - root - INFO - Successfully shut down the VM: VMtest1
2022-05-03 03:19:58 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest1
2022-05-03 03:19:58 - sshInstance.py - [line:54] - root - INFO - Domain VMtest1 has been undefined


2022-05-03 03:20:03 - test_network_sriov_stress.py - [line:350] - root - INFO - Successfully removed the VM: VMtest1
2022-05-03 03:20:06 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest2
2022-05-03 03:20:07 - sshInstance.py - [line:54] - root - INFO - Domain VMtest2 destroyed


2022-05-03 03:20:12 - test_network_sriov_stress.py - [line:340] - root - INFO - Successfully shut down the VM: VMtest2
2022-05-03 03:20:15 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest2
2022-05-03 03:20:15 - sshInstance.py - [line:54] - root - INFO - Domain VMtest2 has been undefined


2022-05-03 03:20:20 - test_network_sriov_stress.py - [line:350] - root - INFO - Successfully removed the VM: VMtest2
.2022-05-03 03:20:20 - test_network_sriov_stress.py - [line:102] - root - INFO - removing VMs: VMtest1, VMtest2
2022-05-03 03:20:23 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest2
2022-05-03 03:20:23 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest2'


2022-05-03 03:20:28 - test_network_sriov_stress.py - [line:340] - root - INFO - Successfully shut down the VM: VMtest2
2022-05-03 03:20:31 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest2
2022-05-03 03:20:31 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest2'


2022-05-03 03:20:36 - test_network_sriov_stress.py - [line:353] - root - ERROR - Failed to removed the VM VMtest2
2022-05-03 03:20:39 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest1
2022-05-03 03:20:39 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest1'


2022-05-03 03:20:44 - test_network_sriov_stress.py - [line:340] - root - INFO - Successfully shut down the VM: VMtest1
2022-05-03 03:20:47 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest1
2022-05-03 03:20:47 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest1'


2022-05-03 03:20:52 - test_network_sriov_stress.py - [line:353] - root - ERROR - Failed to removed the VM VMtest1
2022-05-03 03:20:55 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-05-03 03:20:55 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:21:00 - test_network_sriov_stress.py - [line:107] - root - INFO - try to reset network for enp137s0f0
2022-05-03 03:21:00 - test_network_sriov_stress.py - [line:113] - root - INFO - try to reset network for ubuntu...
2022-05-03 03:21:03 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 0
2022-05-03 03:21:03 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:21:08 - test_network_sriov_stress.py - [line:116] - root - INFO - try to reset network for ens801f0
2022-05-03 03:21:08 - test_network_sriov_stress.py - [line:118] - root - INFO - try to reset network for redhat
2022-05-03 03:21:08 - sshInstance.py - [line:188] - root - INFO - starting to delete network of ens801f0:
2022-05-03 03:21:14 - sshInstance.py - [line:99] - root - INFO - nmcli connection delete ens801f0
Connection 'ens801f0' (c4648774-2579-4f1b-9a03-eb8bc41049fc) successfully deleted.
[root@redhat8 ~]# 
[root@redhat8 ~]# 
2022-05-03 03:21:17 - sshInstance.py - [line:190] - root - INFO - restart networkManager to make setting valid
2022-05-03 03:21:23 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@redhat8 ~]# 
[root@redhat8 ~]# 
2022-05-03 03:21:26 - sshInstance.py - [line:194] - root - INFO - starting to add network of ens801f0:
2022-05-03 03:21:32 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens801f0 ifname enss801f0
Connection 'ens801f0' (47d026f2-e6d1-49d6-9725-9371b58b8b82) successfully added.
[root@redhat8 ~]# 
[root@redhat8 ~]# 
2022-05-03 03:21:35 - sshInstance.py - [line:197] - root - INFO - restart networkManager to make setting valid
2022-05-03 03:21:41 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@redhat8 ~]# 
[root@redhat8 ~]# 
2022-05-03 03:21:47 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-05-03 03:21:47 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:21:52 - test_network_sriov_stress.py - [line:126] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-05-03 03:21:52 - test_network_sriov_stress.py - [line:128] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-05-03/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm2.img-uvm1.img]_03_09_39
2022-05-03 03:21:55 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-05-03 03:21:55 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:22:00 - test_network_sriov_stress.py - [line:130] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-05-03 03:22:03 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/*.txt
2022-05-03 03:22:03 - sshInstance.py - [line:54] - root - INFO - 
2022-05-03 03:22:08 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-05-03 03:22:08 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm2.img-uvm1.img] case ============================
                  [100%]

=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:116
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:116: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:169
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:169: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:102
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:102: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:141
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:141: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:617
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:617: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:744
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:744: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:984
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:984: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_network_sriov_stress.py:1123
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:1123: PytestUnknownMarkWarning: Unknown pytest.mark.nicIperfTcpSUTGuestStress - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicIperfTcpSUTGuestStress

npxtester/Tests/testNics/test_network_sriov_stress.py:1124
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:1124: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:119
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:119: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:156
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:156: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:157
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:157: PytestUnknownMarkWarning: Unknown pytest.mark.connect - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.connect

npxtester/Tests/testNics/test_nics.py:189
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:189: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:190
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:190: PytestUnknownMarkWarning: Unknown pytest.mark.static_ip - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.static_ip

npxtester/Tests/testNics/test_nics.py:221
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:221: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:222
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:222: PytestUnknownMarkWarning: Unknown pytest.mark.test - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.test

npxtester/Tests/testNics/test_nics.py:308
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:308: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:334
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:334: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:377
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:377: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html -
========== 1 passed, 19 deselected, 19 warnings in 765.61s (0:12:45) ===========
[8mha:////4LeiMYyvaTAszhIQ10pdIeDepJ20hGjpoNTSDQ54vTlOAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0SCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQGMKkS1vgAAAA==[0m[Pipeline] }
[8mha:////4I/tRoD/KV6lJ76TTEJ8vusLD5mbZiDJ1z7tZBIwhHThAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKCQKJBqWgtN7zAJMY4se6C7eBUvIiv8QcsIlGxxWpnm3m9YRk81OwNdpp6S6GxOLjR5IWJfX91nLDjCzZMgZ1GqZPkVp9mlBw1zClKWAhYaWocB0smwlp06qEqp8hU5+jzdxRQ2vYOTyhENkfl4xcmD5vb2Aamn+evAKZhiFDut7l2hw9mQDtDvgAAAA==[0m[Pipeline] // stage
[8mha:////4CAP7WQdC2FMpNczqgZSKdPjM+Zu6Lz59I8VEE3/AlUOAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjBB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INOibpDcEAAAA=[0m[Pipeline] stage
[8mha:////4Db8sOlSRxZnVxb7xJBDSAteoF0PgC9lkQldzKIJFwYiAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUwIiYukZZOEFoQkgb/V9+UtqJE3E17kBLJSY8WLYl673eUESGE7ETjcXWY6y96ELvpiQG4vYaaBANXURNGClYoeygyNhqqYqShUVZDisJa4t1oOjRJdjIRj90GTS68px42o4Scm/u8IRMTuSkOX3LyLC99SYS/jh/ATB2CfLDbrb9/Cw+2YMO4MIAAAA=[0m[Pipeline] { (collect logs)
[8mha:////4LN/0LMxu1hyybTBeE6CHwDx82bv/lkvxFk3cdgHTSuWAAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikRJR0VppeIFJjLFj3Tm2g1PxIr7GHwhEomKL1c4283zBOgY4ctBoFfWGYmvQu1HPCzOH/uo4o+ULtkyRncJG5YY7dVqw4aRgSVHCSsBGUes4GtIJtsLKu6ycJF2dU5i/g4DSdAM8oBCzOcmQvjAF2N3GLjL9PH8FMPkEZb3/VO3favo5XL4AAAA=[0m[Pipeline] sh
+ sed -i $'s/\r//g' /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4FgvIflgycWoJq7MsZycbV9d7HENRG2WSX14oF+Me/GuAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiAR2iSmu54QUmNsaJdWdsh6TiRXyNPxCIRMUWq51t5vmCZYpw5GixNdQ5So3D4Hs7LRw4dhfPA7Z8xoYpsTcozSBZm3pGydnAnKKEhYCVocZzcmQzrEWr7qryimx1ynH6DgJKp2/wgEJM5qxi/sIYYXPtdWL6ef4KYAwZyt32U/vwBq1seBO+AAAA[0m[Pipeline] sh
+ chmod a+x /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4P37N80+NRbzyFlga2j1WDon6H/4VmnQwrRysqQpQX/3AAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigagQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUO62n9qHN+xdYwq+AAAA[0m[Pipeline] sh
+ sh /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4P3ih/vRXCzaXCkS1CQDBQXrb3I4EZcCr0NGGH+tKlR4AAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigUSDqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFDu9p/avgEjVhSLvgAAAA==[0m[Pipeline] }
[8mha:////4ISV0Ptmba0km8a80Gq+UClogdZnPDxCJXroOoeI5sULAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiREGBqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFDu9p/avgE81EHZvgAAAA==[0m[Pipeline] // stage
[8mha:////4LmA5LOJ1yGOO5YQ6RYn+/szIsYEkfxAArsUKSFs0Xa0AAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIpoEBCVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5Nu9INt9AE/B/a+9AAAA[0m[Pipeline] }
[8mha:////4KVKO0UZMawo8C9tjXN4L1+ryWbU4zmvD2sDlsKoyYtXAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiIUVI5jZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0gL/eCrPwAXH35DL0AAAA=[0m[Pipeline] // node
[8mha:////4A8vvhfR6oegt+2XAbvn3Z0Fx6j3lb1QUh2abilCs+OGAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMJBiA5R0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77eCbPcBAx/ra70AAAA=[0m[Pipeline] End of Pipeline
Finished: SUCCESS
